test/unit/protocol/StreamrBinaryMessage.test.js

```diff
@@ -2,6 +2,9 @@
 const assert = require('assert')
 const sinon = require('sinon')
 const StreamrBinaryMessage = require('../../../src/protocol/StreamrBinaryMessage')
+const StreamrBinaryMessageV28 = require('../../../src/protocol/StreamrBinaryMessageV28')
+const StreamrBinaryMessageV29 = require('../../../src/protocol/StreamrBinaryMessageV29')
```

Comment:
Would be great to split this file to:
- `StreamrBinaryMessageV28.test.js` (test version 28, basically contents of `describe('version 28') block`)
- `StreamrBinaryMessageV29.test.js` (test version 29, basically contents of `describe('version 29') block`)
- `StreamrBinaryMessageFactory.test.js` (test that `fromBytes` outputs the correct class with something like `assert(StreamrBinaryMessageFactory.fromBytes(msgV28.toBytes()) instanceof StreamrBinaryMessageV28)`)
- `StreamrBinaryMessage.test.js` (maybe not needed: no code to test here anymore?)